src/pages/billing/Project.js

```diff
@@ -106,17 +127,65 @@ const WorkspaceCard = Utils.memoWithName('WorkspaceCard', ({ workspace, isExpand
           ])
         ])
       ]),
-      isExpanded && div({ id, style: workspaceCardStyles.row }, [
+      isExpanded && div({ id, style: { ...workspaceCardStyles.row, padding: '0.5rem', border: `1px solid ${colors.light()}` } }, [
         div({ style: workspaceCardStyles.expandedInfoContainer }, [
           h(ExpandedInfoRow, { title: 'Google Project', details: googleProject }),
-          h(ExpandedInfoRow, { title: 'Billing Account', details: billingAccountName })
+          h(ExpandedInfoRow, { title: 'Billing Account', details: billingAccount, errorMessage: billingAccountErrorMessage })
         ])
       ])
     ])])
   ])
 })
 
-const ProjectDetail = ({ project, project: { projectName, creationStatus }, billingAccounts, authorizeAndLoadAccounts }) => {
+const BillingAccountSummaryPanel = (() => {
+  const StatusAndCount = ({ status, count }) => div({ style: { display: 'float' } }, [
+    div({ style: { float: 'left' } }, [getBillingAccountIcon(status)]),
+    div({ style: { float: 'left', marginLeft: '0.5rem' } }, [`${status} (${count})`])
+  ])
+
+  const maybeAddStatus = (status, count) => count > 0 && div({ style: { marginRight: '2rem' } }, [
+    h(StatusAndCount, { status, count })
+  ])
+
+  return Utils.memoWithName(
+    'BillingAccountSummaryPanel',
+    ({ counts: { done, error, updating } }) => div({
+      style: {
+        padding: '0.5rem 2rem 1rem',
+        position: 'absolute',
+        top: topBarHeight,
+        right: '3rem',
+        width: '30rem',
+        backgroundColor: colors.light(0.5),
+        boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)'
+      }
+    }, [
+      div({ style: { padding: '1rem 0' } }, 'Your billing account is updating...'),
+      div({ style: { display: 'flex', justifyContent: 'flex-start' } }, [
+        maybeAddStatus('updating', updating),
+        maybeAddStatus('done', done),
+        maybeAddStatus('error', error)
+      ]),
+      error > 0 && div({ style: { padding: '1rem 0 0' } }, [
+        'Try again or ',
+        h(Link, { onClick: () => contactUsActive.set(true) },
+          ['contact us regarding unresolved errors']),
+        '.'
+      ])
+    ]))
+})()
+
+const groupByBillingAccountStatus = (billingProject, workspaces) => {
+  const group = workspace => Utils.cond(
+    [billingProject.billingAccount === workspace.billingAccount, () => 'done'],
+    ['billingAccountErrorMessage' in workspace, () => 'error'],
```

Comment:
We don't use `in` in this codebase, instead consider `!!workspace.billingAccountErrorMessage` or `_.has`